.gitignore

```diff
@@ -50,3 +50,6 @@ npm-debug.log
 
 # Vim
 *.swp
+
+# VS Code
+.vscode
```

Comment:
I would add this to your own global gitignore instead of to this project specific one. That being said, since there's already vim, sublime, and other editors, it might be worth making a separate PR to add VS Code to the Dillinger gitignore